testing/tests/DevExpress.exporter/jspdfParts/jspdfTestHelper.js

```diff
@@ -92,4 +92,34 @@ class JSPdfDataGridTestHelper {
     }
 }
 
-export { JSPdfDataGridTestHelper };
+class JSPdfPrintInfoCalculatorTestHelper {
+    constructor(jsPDFDocument) {
+        this.jsPDFDocument = jsPDFDocument;
+    }
+
+    checkColumnWidths(expectedColumnWidths, options) {
+        const columnStyles = options.columnStyles;
+        const actualWidths = [];
+
+        for(let i = 0; i < expectedColumnWidths.length; i++) {
+            const width = columnStyles[i] ? columnStyles[i].width : 0;
```

Comment:
For a separate TablePrinter (width === undefined/null) should mean 'auto'.
IMHO we should change this.
And, it' not clear which class should make this choice: calculator or printer?